README.md

```diff
@@ -7,31 +7,53 @@ can then perform whatever actions it wants.
 
 ## Basic usage
 
-`snare` requires three command-line arguments to be specified:
+`snare` has the following command-line format:
 
 ```
-Usage: snare [-e email] [-j <max-jobs>] -p <port> -r <repos-dir> -s <secrets-path>
+Usage: snare [-c <config path>]
 ```
 
 where:
 
- * `<email>` is an (optional) email address to which any errors running per-repo
-   programs will be sent (warning: full stderr/stdout will be sent, so consider
-   carefully whether these have sensitive information or not).
- * `<max-jobs>` is an (optional) non-zero integer specifying the maximum number
-   of jobs to run in parallel.
- * `<port>` is a port number (e.g. 4567).
- * `<repo-programs-dir>` is the directory where the per-repo programs are
+ * `<config path>` is a path to a `snare.conf` configuration file. If not
+   specified explicitly, the following locations will be searched, in order:
+     * `/etc/snare.conf/`
+     * `~/.snare.conf`
+
+
+## Configuration file
+
+The configuration file supports the following options:
+
+ * `email = "<address>"` optionally specifies an email address to which any
+   errors running per-repo programs will be sent (warning: full stderr/stdout
+   will be sent, so consider carefully whether these have sensitive information
+   or not).
+ * `maxjobs = <int>` is an (optional) non-zero integer specifying the maximum
+   number of jobs to run in parallel. Defaults to the number of CPUs in the
+   machine.
+ * `port = <int>` is a mandatory port number to listen on (e.g. 4567).
+ * `reposdir = "<path>"` is the directory where the per-repo programs are
    stored. For a repository `repo` owned by `user` the command
-   `<repo-programs-dir>/<user>/<repo> <event> <path to GitHub JSON>` will be
-   run. Note that per-repo programs are run with their current working
-   directory set to a temporary directory to which they can freely write and
-   which will be automatically removed when they have completed.
- * `<secrets-path>` is the file containing the GitHub secret which guarantees
-   that hooks are coming from your GitHub repository and not a malfeasant. Note
-   that leading and trailing whitespace in this file is ignored.
-
-The most important part of this are the per-repo programs. For example,
+   `<reposdir>/<user>/<repo> <event> <path to GitHub JSON>` will be run. Note
+   that per-repo programs are run with their current working directory set to a
+   temporary directory to which they can freely write and which will be
+   automatically removed when they have completed.
+ * `secret = "<secret>"` is the mandatory GitHub secret which guarantees that
+   hooks are coming from your GitHub repository and not a malfeasant.
+
+The minimal configuration file is thus:
+
+```
+port = <port>
+reposdir = "<path>"
+secret = "<secret>"
+```
+
+
+## Per-repo programs
+
+The most important part of snare are the per-repo programs. For example,
```

Comment:
> the most important part of snare are

Should it be:

> the most important part of a snare deployment are

As the programs are not part of snare, per-se, right? The user provides those.

Reply:
Fixed in https://github.com/softdevteam/snare/pull/12/commits/0cf50f27281b617a03b5b5f97503de499429e5f9.